### PR TITLE
feat(unlock-app) - replace jest with vitest in unlock-app

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/__tests__/components/page/OpenGraphTags.test.js [hunk 1]
```diff
@@ -1,68 +0,0 @@
-import React from 'react'
```
Comment (julien51) [inline unlock-app/src/__tests__/components/page/OpenGraphTags.test.js:1]: Arre we not using this anymore?

**Review by julien51 — APPROVED** [review]
I am surprised coverage does not change with all the tests removed, but I guess this is good enough?

File: unlock-app/package.json [hunk 2]
```diff
@@ -96,9 +85,14 @@
   "devDependencies": {
     "@babel/plugin-proposal-optional-chaining": "7.20.7",
     "@unlock-protocol/tsconfig": "workspace:./packages/tsconfig",
+    "@vitejs/plugin-react": "^3.0.0",
```
Comment (searchableguy) [inline unlock-app/package.json:88]:
```suggestion
    "@vitejs/plugin-react": "3.0.0",
```

**Review by searchableguy — APPROVED** [review]
LGTM

**Pull request merged**